draft-ietf-scitt-architecture.md

```diff
@@ -228,9 +228,8 @@ Over time, an Issuer may register new Signed Statements about an Artifact in a T
 Receipt:
 
 : a cryptographic proof that a Signed Statement is included in the Append-only Log.
-Receipts are based on Signed Inclusion Proofs, such as those as described in COSE Signed Merkle Tree Proofs {{-COMETRE}};
-they can be built on different verifiable data structures, not just binary merkle trees.
-A Receipt consists of a Transparency Service-specific inclusion proof for the Signed Statement, a signature by the Transparency Service of the state of the Append-only Log after the inclusion, and additional metadata (contained in the signature's protected headers) to assist in auditing.
+Receipts are signed proofs of verifiable data-structure properties.
```

Comment:
By removing the reference to [the draft-ietf-cose-merkle-tree definition of receipti](https://www.ietf.org/archive/id/draft-ietf-cose-merkle-tree-proofs-06.html#name-terminology), are we separating the definition in SCITT as unique? If so, for what purpose?